core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/output/ToCsvHandler.java

```diff
@@ -73,15 +74,18 @@ private LinkedHashMap<String, String> getPropertyHeadersFromSchema(final Store s
             }
         }
         for (final SchemaEdgeDefinition schemaEdgeDefinition : schema.getEdges().values()) {
-            for (final IdentifierType identifierType:schemaEdgeDefinition.getIdentifiers()) {
+            for (final IdentifierType identifierType : schemaEdgeDefinition.getIdentifiers()) {
                 if (identifierType.toString().equals(identifierType.DIRECTED.toString())) {
```

Comment:
This should also be improved. There's no reason to convert this all to strings and then compare. This can be done instead:
```suggestion
                if (identifierType.equals(IdentifierType.DIRECTED)) {
```

Reply:
Have implemented your suggestion